.idea/fileTemplates/includes/File Header.java

```diff
@@ -0,0 +1,14 @@
+/**
+ *    Copyright ${YEAR} DuckDuckGo
```

Comment:
Looks good! Remove the additional space at the start of this line and add a blank line underneath to match the header recommendation at http://www.apache.org/licenses/LICENSE-2.0.html#apply.